opensrp-reveal/src/main/java/org/smartregister/reveal/fragment/TaskRegisterFragment.java

```diff
@@ -105,6 +110,21 @@ public void setupViews(View view) {
         drawerView.onResume();
 
         initializeProgressIndicatorViews(view);
+
+        view.findViewById(R.id.filter_text_view).setOnClickListener(v -> {
+            startActivityForResult(new Intent(getContext(), FilterTasksActivity.class), org.smartregister.reveal.util.Constants.RequestCode.REQUEST_CODE_FILTER_TASKS);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unnecessary use of fully qualified name 'org.smartregister.reveal.util.Constants.RequestCode.REQUEST_CODE_FILTER_TASKS' due to existing static import 'org.smartregister.reveal.util.Constants.RequestCode.REQUEST_CODE_FILTER_TASKS'](https://app.codacy.com/manual/OpenSRP/opensrp-client-reveal/pullRequest?prid=4684568)